src/components/DiagramFrame/SeqDiagram/LifeLineLayer/Participant.vue

```diff
@@ -37,8 +37,8 @@ import { computed, ref } from 'vue';
 import useDocumentScroll from '@/functions/useDocumentScroll';
 import useIntersectionTop from '@/functions/useIntersectionTop';
 import { useStore } from 'vuex';
+import { getElementDistanceToTop } from '@/utils/dom';
 
-const PARTICIPANT_TOP = 50  // the distance of participants from the top of the diagram
 const PARTICIPANT_HEIGHT = 70
 const INTERSECTION_OFFSET = 10 // a threshold for judging whether the participant is intersecting with the viewport
```

Comment:
rename to `INTERSECTION_THRESHOLD`?

Reply:
The IntersectionObserver API already have a `threshold` parameter, a variable called `INTERSECTION_THRESHOLD ` may causing confusion, I will renamed it to `INTERSECTION_ERROR_MARGIN`